api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -633,3 +637,17 @@ message HttpFilter {
     google.protobuf.Any typed_config = 4;
   }
 }
+
+message RequestIDUtils {
+  reserved 3;
+
+  // The name of the request id utilities extension to create.
+  string name = 1 [(validate.rules).string = {min_bytes: 1}];
+
+  // Request ID utility specific configuration.
+  oneof config_type {
+    google.protobuf.Struct config = 2 [deprecated = true];
```

Comment:
Please remove both name and config, and just use typed_config to load the extension.